<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminsPanel</class>
 <widget class="QDialog" name="adminsPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Dodaj nowy film</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Nazwa filmu</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="movieNameLE"/>
   </item>
   <item>
    <widget class="QPushButton" name="addMovieBtn">
     <property name="text">
      <string>Dodaj Film</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Dodaj nowy seans</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Film</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="movieCB"/>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Wprowadź datę seansu</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="dateShowLE"/>
   </item>
   <item>
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Podaj godzinę seansu</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="timeShowLE"/>
   </item>
   <item>
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Wybierz salę</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="halliDCB"/>
   </item>
   <item>
    <widget class="QPushButton" name="addShowBtn">
     <property name="text">
      <string>Dodaj seans</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
